src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Square extends Figure implements AreaCalculetion {
+    private double side;
+
+    public Square(double side, String color, String name) {
+        this.side = side;
+    }

```

Comment:
The 'color' parameter is not being used to set the color of the square. You should assign this to the superclass property using 'super(color, name)'.